### ISSUE TITLE
Is this project abandoned?

### ISSUE THREAD

**Issue by spmsupun** [mailcheck/mailcheck#187]
Is this project abandoned? If you guys need any contributors I can help.

**Comment by ferreiro** [comment]
Yes @spmsupun, this library is no longer maintained (7 years since the last update), also contains a couple of bugs like [this](https://github.com/ZooTools/email-spell-checker/pull/3) or [this](https://github.com/ZooTools/email-spell-checker/pull/4)

I rewrote it entirely and update it for 2022 and onward, open to get contributors: https://github.com/ZooTools/email-spell-checker

- 💙 Written in TypeScript and removed jQuery
- 🚀 Reduced bundle size to <2KB. 
- ✨ Update TLDs (69+) and added modern startup domains (like .io, .so, .xyz or .dev)
- 🙏 Implemented suggestions that people made in this repo that were ignored.

Link: https://github.com/ZooTools/email-spell-checker

Come check it out and give it a ⭐️ for the effort.

**Comment by ferreiro** [comment]
cc: @gdvalderrama @jemarjones @robsonsobral @m-verlund @noinkling @daijobou @gnoesiboe @sarah-ifg @ruanitto (u guys reacted with a thumbs up emoji)

**Comment by Chrysweel** [comment]
Hi everyone, It looks like the project continue inactive, there is a recent proposal in Issue #194  from @rokkoo, who offered to help maintain the repository.

It would be great if the current maintainers could take a look and see if handing over some permissions is an option to keep the project alive.

**Comment by skyfallsin** [comment]
Mailcheck is actively maintained again. The repository now has current CI and a 2.0 prerelease build in progress; we will share the prerelease as soon as the release work is ready. Thanks for checking in.
